helm_deploy/values-prod.yaml

```diff
@@ -74,7 +74,7 @@ variables:
   environment: production
   portalIDPMetadataFile: config/laa_portal/metadata/prod.xml
   spEntityId: LAA_Portal_CRM7_Prod
-  appStore: https://laa-crime-application-store-prod.apps.live.cloud-platform.service.justice.gov.uk
+  appStore: http://laa-crime-application-store-app.laa-crime-application-store-production.svc.cluster.local
```

Comment:
@ivanELEC does it matter that this is not https?

Reply:
So my assumption is that because this can only ever possibly be accessed internally that we are secured from outside noseying - this is also how LGA have it setup (all http not https)